.gitignore

```diff
@@ -2,3 +2,4 @@
 .classpath
 .project
 .settings
+/spec/target
```

Comment:
A simpler rule of ...

```
target/
```

will ignore all of the target directories, in all maven projects.

Reply:
Updated.